lib/dat-worker-pool/worker.rb

```diff
@@ -87,6 +87,10 @@ def raise(*args)
         @thread.raise(*args) if self.thread_alive?
       end
 
+      # Helpers - TODO make these private once callbacks are instance evald
+      def params; @runner.worker_params; end
```

Comment:
@jcredding cool use of the runner to store the params once instead of duplicate storing them on each worker!  nice.

well, as I write this I'm wondering, does this mean the params have to be thread safe now?  does the runner need to mutex lock accessing the params?  thoughts on this?  should we just let them hang themselves if them use threads for anything other than read-only stuff?